src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.ComposeCache.targets

```diff
@@ -35,14 +145,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     Prepare the prerequisites for ComposeCache.
     ============================================================
     -->
+  
   <Target Name="PrepareForComposeCache">
 
     <PropertyGroup>
       <PreserveCacheLayout Condition="'$(PreserveCacheLayout)' == ''">true</PreserveCacheLayout>
       <SkipOptimization Condition="'$(RuntimeIdentifier)' == ''">true</SkipOptimization>
```

Comment:
This should check if someone else already set `SkipOptimization`, and not overwrite what a user set.